coalib/bears/Bear.py

```diff
@@ -130,6 +130,12 @@ class Bear(Printer, LogPrinterMixin):
     >>> class SomeBear(Bear): pass
     >>> SomeBear.source_location
     '...Bear.py'
+
+    Every lint bear makes use of an executable tool for its operations.
+    The SEE_MORE attribute provides a link to the main page of the tool:
+
+    >>> class SomeBear(Bear):
```

Comment:
The rest of the examples in the docstring use 
`class SomeBear(Bear): pass` 
So I thought of using the same.